lib/matplotlib/tests/test_collections.py

```diff
@@ -1149,3 +1149,40 @@ def test_check_masked_offsets():
 
     fig, ax = plt.subplots()
     ax.scatter(unmasked_x, masked_y)
+
+
+@check_figures_equal(extensions=["png"])
+def test_masked_set_offsets(fig_ref, fig_test):
+    x = np.ma.array([1, 2, 3, 4, 5], mask=[0, 0, 1, 1, 0])
+    y = np.arange(1, 6)
+
+    ax_test = fig_test.add_subplot()
+    scat = ax_test.scatter(x, y)
+    scat.set_offsets(np.ma.column_stack([x, y]))
+    ax_test.set_xticks([])
+    ax_test.set_yticks([])
+    ax_test.set_xlim(0, 7)
+    ax_test.set_ylim(0, 6)
```

Comment:
I would have assumed the autoscaling and tick setting would also be independent of whether MaskedArrays were used or not... Do you need to explicitly set these bounds? Otherwise I'd say leave them in as that is another sanity check on the implementation.

Reply:
I was setting the limits because I was facing some issues earlier when I changed the values before setting them. Since I'm not doing them anymore, that should be fine. I removed the ticks mainly to not have text.